packages/bpk-component-select/src/BpkSelect.js

```diff
@@ -30,6 +30,7 @@ export type Props = {
   name: string,
   value: string,
   className: ?string,
+  disabled: ?boolean,
```

Comment:
I think this should just be `boolean`, as it isn't possible for it to be `null` due to having a defaultProp of `false`. Same as with `docked` etc.